# Generated for NaoQi method ALLaser.laserOFF
# Disable laser light

---


